os/drivers/audio/alc5658scripts.h

```diff
@@ -29,370 +29,281 @@
-	{ALC5658_0x01DE, 0x4500, 0},
-	{ALC5658_0x01E4, 0x001F, 0},
-	{ALC5658_0x01DE, 0xC500, 600},
-	{ALC5658_0x01E4, 0x0000, 0},
-	{ALC5658_0x01DF, 0x20C0, 0},
+	{ALC5658_MANAGEMENT5, 0x0184, 0},	/* ???NonDescribed,  LDO2 ON */
+	{ALC5658_013C, 0x3C05, 0},
+	{ALC5658_01DF, 0x2cc1, 6},
+	{ALC5658_01DE, 0x5100, 6},
+	{ALC5658_01E4, 0x0014, 6},
+	{ALC5658_01DE, 0xd100, 30},
+	{ALC5658_01DF, 0x20C1, 0},
+	{ALC5658_01DF, 0x2CC1, 0},
+	{ALC5658_01DE, 0x4900, 0},
+	{ALC5658_01E4, 0x0016, 0},
+	{ALC5658_01DE, 0xC900, 250},
+	{ALC5658_01DF, 0x2CC1, 0},
+	{ALC5658_HPOUT, 0x0000, 0},	/* UNMUTE HP Output */
+	{ALC5658_01DE, 0x4500, 0},
+	{ALC5658_01E4, 0x001F, 0},
+	{ALC5658_01DE, 0xC500, 600},
+	{ALC5658_01E4, 0x0000, 0},
+	{ALC5658_01DF, 0x20C0, 0},
 	{ALC5658_0040, 0x0808, 0},
-	{ALC5658_PWR_MNG5, 0x0000, 0},	/* LDO2 OFF */
-	{ALC5658_0x013C, 0x2005, 0},
-	{ALC5658_PWR_MNG6, 0x0000, 0},	/* MIXERs OFF */
-	{ALC5658_PWR_MNG7, 0x0000, 0},	/* VOL Out/In ctrl, MIC in det OFF */
-	{ALC5658_PWR_MNG1, 0x0000, 0},	/* I2S, DAC, ADC, SPDIFF, CLASS D OFF */
-	{ALC5658_PWR_MNG2, 0x0000, 0},	/* Filters, PDM I/F  OFF */
-	{ALC5658_PWR_MNG3, 0x003C, 0},	/* HP L/R ON, HPamp x5, LDO1 out 0.9V */
-	{ALC5658_GLBL_CLK, 0x0000, 0},	/* MCLK, No dividers */
-	{ALC5658_ADDA_CLK, 0x0770, 0},	/* I2Sprediv1 = 2, bclk_ms2 = 16bits(32FS), I2Sprediv2 = 7, something reserved, dac/adc 128Fs */
-	{ALC5658_INTCLK_CTRL, 0x0080, 0},	/* Probably enable Pow_int_clk1/2, does not match with DOC !!! */
+	{ALC5658_MANAGEMENT5, 0x0000, 0},	/* LDO2 OFF */
+	{ALC5658_013C, 0x2005, 0},
+	{ALC5658_MANAGEMENT6, 0x0000, 0},	/* MIXERs OFF */
+	{ALC5658_MANAGEMENT7, 0x0000, 0},	/* VOL Out/In ctrl, MIC in det OFF */
+	{ALC5658_MANAGEMENT1, 0x0000, 0},	/* I2S, DAC, ADC, SPDIFF, CLASS D OFF */
+	{ALC5658_MANAGEMENT2, 0x0000, 0},	/* Filters, PDM I/F  OFF */
+	{ALC5658_MANAGEMENT3, 0x003C, 0},	/* HP L/R ON, HPamp x5, LDO1 out 0.9V */
+	{ALC5658_CLOCK, 0x0000, 0},	/* MCLK, No dividers */
+	{ALC5658_ADCDAC_CLOCK, 0x0770, 0},	/* I2Sprediv1 = 2, bclk_ms2 = 16bits(32FS), I2Sprediv2 = 7, something reserved, dac/adc 128Fs */
+	{ALC5658_MICBIAS2, 0x0080, 0},	/* Probably enable Pow_int_clk1/2, does not match with DOC !!! */
 };
 
 t_codec_init_script_entry codec_init_out_script[] = {
-	{ALC5658_RESET, 0x0000, 0},
+	{ALC5658_SW_RESET, 0x0000, 0},
 	{ALC5658_006E, 0xFFFF, 0},
 	{ALC5658_006F, 0xFFFF, 0},
-	{ALC5658_GNRL_CTRL, 0x8001, 0},	/* Bypass_noise_gate_mode2 BYPASS, digital_gate_ctrl ENABLE */
-	{ALC5658_ADDA_RST1, 0x3030, 0},	/* ??? DAC1, DAC2 + alpha clock enable */
-	{ALC5658_GLBL_PLL1, 0x0302, 0},	/* k = 2, n = 8 */
-	{ALC5658_GLBL_PLL2, 0x0800, 0},	/* m - bypass */
-	{ALC5658_ADDA_CLK, 0x1110, 0},	/* I2Sprediv1 = 2, bclk_ms2 = 16bits(32FS), I2Sprediv2 = 2, something reserved, dac/adc 128Fs */
+	{ALC5658_GENERAL_CONTROL_1, 0x8001, 0},	/* Bypass_noise_gate_mode2 BYPASS, digital_gate_ctrl ENABLE */
+	{ALC5658_RESET1, 0x3030, 0},	/* ??? DAC1, DAC2 + alpha clock enable */
```

Comment:
what is "???" ?
plese check and then modify that comment.

Reply:
@jsdosa I did not change the comment in the script. I do not know the exact meaning of script comments. Sorry.